### ISSUE TITLE
Super helpful guide. Questions.

### ISSUE THREAD

**Issue by octopickle** [olm3ca/mirror#3]
Thanks for the detailed guide! 
A few questions: 
Does [this vizio repair board kit](https://www.ebay.com/itm/255281574129?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=PzmB0XZKRQG&sssrc=2047675&ssuid=9JWmdxBASDW&widget_ver=artemis&media=COPY) seem like it has the required cables? 
The repair kit above comes with a TCON board, does that get added to the mirror as well?
Does one NEED to add a pi or computer board as well, or will an HDMI cable attached to the tv board suffice for an input?

**Comment by olm3ca** [comment]
That's the exact repair kit I ordered, you should be good to go! You don't need the TCON board - your mirror already has the exact same one. And no, you don't need a Pi. Just an HDMI cable attached to whatever device you want to plug into it will work. Just remember that whatever device you use, you'll need to rotate the screen 90 degrees because the TV has no setting to run in portrait mode. Any PC or Mac can handle that, but not Fire sticks / chromecasts, etc. 

One other minor item I found useful, at least during setup, was this [$6 replacement remote control](https://www.amazon.com/dp/B07L8BMYHJ?psc=1&ref=ppx_yo2ov_dt_b_product_details) to access the TV's settings and add it to wifi, etc.

**Comment by pwatkinson6** [comment]
Thanks for doing this and your comprehensive guide!! This is fantastic! I am not a very tech savvy person but am excited to give this a go. 

1. Is this the correct Vizio repair kit (looked similar to the octopickle): https://www.ebay.com/itm/254635079275?hash=item3b496ef66b:g:tkUAAOSwxppe9NDk ?

2. Is there anything specific I should know or do for the Raspberry Pi setup besides what you explained in the article? I have found a few videos on setting up a Raspberry Pi and hopefully that will help.

**Comment by olm3ca** [comment]
@pwatkinson6 that is the correct board, yes. As for anything Pi related, not really - you'll want to set up MagicMirror which is very well documented, and I'd suggest installing uxplay if you want it to behave like an Airplay / bluetooth speaker to stream music to. Good luck!

**Comment by octopickle** [comment]
@pwatkinson6 One thing I found that is needed in addition was a power cable for the Vizio board - [this one](https://www.ebay.com/itm/274670237923) did the trick. I got everything up and running as a display with audio. 
Still trying to figure out how I ultimately want to use this thing. I'm currently messing around with some stuff on a Raspberry Pi, I'll post an update when I decide what to do with it. 
Good luck!
